Makefile

```diff
@@ -0,0 +1,32 @@
+DESTDIR ?= /
```

Comment:
Unused. We need it in all install/uninstall commands.

Reply:
What do you mean? Should I add that as a comment?

Reply:
It is used. Check the next two lines.

Reply:
Ouch! Sorry.

I expected it to be used in every single `install` call so I somewhat missed the two following lines :sweat_smile: